From d76f525e3e443c6697bb63bccfc4e24b33b96ef1 Mon Sep 17 00:00:00 2001
From: Rajesh Bhagat <rajesh.bhagat@nxp.com>
Date: Fri, 2 Nov 2018 16:43:53 +0530
Subject: [PATCH 343/706] arm64: dts: ls1028a: Add fspi and dspi nodes for simu
 platform

Adds fspi and dspi nodes for simu platform

Signed-off-by: Rajesh Bhagat <rajesh.bhagat@nxp.com>
(cherry picked from commit eae3e766f0ba6823a8ea37c2bd2321f41c46caa2)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 .../boot/dts/freescale/fsl-ls1028a-simu.dts   | 23 ++++++++++++++++++-
 1 file changed, 22 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a-simu.dts b/arch/arm64/boot/dts/freescale/fsl-ls1028a-simu.dts
index 10459694c0b6..11b7e2f654f3 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a-simu.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a-simu.dts
@@ -1,7 +1,7 @@
 /*
  * Device Tree file for LS1028A software Simulator model
  *
- * Copyright 2017, NXP
+ * Copyright 2017-2018, NXP
  *
  * Sudhanshu Gupta <sudhanshu.gupta@nxp.com>
  *
@@ -125,3 +125,24 @@
 		full-duplex;
 	};
 };
+
+&fspi {
+	status = "okay";
+	flash0: mt35xu02g@0 {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "micron,m25p80";
+		m25p,fast-read;
+		spi-max-frequency = <20000000>;
+		reg = <0>;
+		/* The following setting enables 1-1-8 (CMD-ADDR-DATA) mode */
+		spi-rx-bus-width = <8>; /* 8 SPI Rx lines */
+		spi-tx-bus-width = <1>; /* 1 SPI Tx line */
+	};
+};
+
+&dspi0 {
+	bus-num = <0>;
+	status = "okay";
+
+};
-- 
2.17.1

